app/views/layouts/_error_messages.html.erb

```diff
@@ -0,0 +1,20 @@
+<%# @details={:title=>[{:error=>:blank}]},
+@messages={:title=>[" must be given"]}> %>
+
+<% if model.errors.messages.length > 0 %>
```

Comment:
You probably don't need both this partial and the section in `application.html.erb` to display flash errors. Pick one error reporting technique and stick with it.